release_notes/Service_Fabric_ReleaseNotes_90.md

```diff
@@ -0,0 +1,153 @@
+| 10. | **Feature** | Added Fabric_InstanceId environment variable for stateless guest applications | **Brief Description**: Adding Fabric_InstanceId environment variable for stateless guest applications.<br>**Impact**: In order to report health for guest applications replicas using HTTP-based Fabric client from within the guest application, the following information is required PartitionId, ReplicaId (for stateful replicas) and InstanceId (for stateless replicas).<br>Today we already provide PartitionId and ReplicaId (for stateful replicas) as environment variables. Adding the missing InstanceId environment variable for stateless replicas will complete the required information.<br>**Documentation Reference**: To view the list of available Service Fabric environment variable - <ul><li>[Service Fabric environment variables](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-environment-variables-reference)</li></ul> |
+| 11. | **Feature** | Introduced a property in Service Fabric runtime that can be set via SFRP as the ARM resource id | **Brief Description**: To better support Service Fabric Resource Provider (SFRP) & Service Fabric managed clusters (SFMC) scenarios, SF runtime needs to allow support for additional metadata for ARM deployed resources.<br>**Impact**: Metadata added for application type are application type provision timestamp and arm resource id. Metadata added for application, service, and cluster are ARM resource id.<br>Existing API's have been updated to read this metadata and support for Powershell, REST, and FabricClient have been added.<br>The runtime created a new API for Service Fabric Resource Provider (SFRP) for updating the metadata. Currently no public support is provided for Powershell since this is only available only for Service Fabric Resource Provider (SFRP) & Service Fabric managed clusters (SFMC) use<br>**Documentation Reference**: Read more about it at - <ul><li>ApplicationTypeMetadata</li><li>ApplicationMetadata</li><li>ServiceMetadata</li></ul> |
+| 12. | **Feature** | Exposed application type provision timestamp | **Brief Description**: Added application type provision timestamp that is exposed via existing application type API's.<br>**Impact**: To better support Service Fabric Resource Provider (SFRP) and Service Fabric managed clusters (SFMC) scenarios, SF runtime needs to allow support for additional metadata for ARM deployed resources.<br>**Documentation Reference**: Read more about it at - <ul><li>ApplicationTypeMetadata</li></ul> |
```

Comment:
Can customers fetch this timestamp to determine when their application deployments completed?